### PR TITLE
New parachain guide

### PR DESCRIPTION
**Pull request by PierreBesson**
Complete revamp of the guide to focus more on a parachain deployment guide.

### CODE REVIEW

File: src/explanations/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+It is highly recommended that projects deploying networks have an infrastructure as code mindset. The infrastructure components in deployments should be written in code and put in a version control system.
+
+This ensures that:
+
+- Your Infrastructure is in a known state
+- Version control is available to see changes and enforce reviews etc...
+- Rolling back is made very easy
+- There are no manual actions required to redeploy
+
+![workflow](images/3osxC7.png)
```
Comment (BulatSaif) [inline src/explanations/index.md:14]: link is broken

File: src/explanations/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+One of the main desired properties of a blockchain is security, so having a very ‘secure by default’ approach to your infrastructure should always be in the forefront of your mind when deploying new networks.
+
+On the infrastructure side, having strict rules and blocking any non-required communication is just a starting point. Other things that should be considered are things like log monitoring, bastion hosts being required for access, 2FA for ssh etc…
+
+On the chain operations side, there are two very useful concepts to increase your security.
+
+First is using cold / warm / hot keys, so that the compromise of say a session key would not lead to any funds being stolen. Also using multisig accounts on offline devices can greatly help security and also give an on-chain audit trail of all actions, allows for alerting for any onchain actions etc...
+
+![key stages](images/AEEVr6.png)
```
Comment (BulatSaif) [inline src/explanations/index.md:26]: link is broken

File: src/explanations/index.md (hunk 1 above)
Comment (PierreBesson) [reply src/explanations/index.md]: fixed

File: src/explanations/index.md (hunk 2 above)
Comment (PierreBesson) [reply src/explanations/index.md]: fixed

**Review by BulatSaif — APPROVED** [review]

**Pull request merged**